tests/integration-tests/configs/common.jinja2

```diff
@@ -5,13 +5,13 @@
 {%- set SCHEDULERS_ALL = ["slurm", "sge", "torque", "awsbatch"] -%}
 {%- set SCHEDULERS_TRAD = ["slurm", "sge", "torque"] -%}
 {%- set OSS_BATCH = ["alinux2"] -%}
-{%- set OSS_COMMERCIAL_X86 = ["alinux2", "centos7", "centos8", "ubuntu1804"] -%}
-{%- set OSS_CHINA_X86 = ["alinux2", "ubuntu1804"] -%}
-{%- set OSS_GOVCLOUD_X86 = ["alinux2", "ubuntu1804"] -%}
-{%- set OSS_COMMERCIAL_ARM = ["alinux2", "centos8", "ubuntu1804"] -%}
-{%- set OSS_CHINA_ARM = ["alinux2", "ubuntu1804"] -%}
-{%- set OSS_GOVCLOUD_ARM = ["alinux2", "ubuntu1804"] -%}
-{%- set OSS_ONE_PER_DISTRO = ["centos7", "alinux2", "ubuntu1804"] -%}
+{%- set OSS_COMMERCIAL_X86 = ["alinux2", "centos7", "centos8", "ubuntu1804", "ubuntu2004"] -%}
+{%- set OSS_CHINA_X86 = ["alinux2", "ubuntu1804", "ubuntu2004"] -%}
+{%- set OSS_GOVCLOUD_X86 = ["alinux2", "ubuntu1804", "ubuntu2004"] -%}
+{%- set OSS_COMMERCIAL_ARM = ["alinux2", "centos8", "ubuntu1804", "ubuntu2004"] -%}
+{%- set OSS_CHINA_ARM = ["alinux2", "ubuntu1804", "ubuntu2004"] -%}
+{%- set OSS_GOVCLOUD_ARM = ["alinux2", "ubuntu1804", "ubuntu2004"] -%}
+{%- set OSS_ONE_PER_DISTRO = ["centos7", "alinux2", "ubuntu1804", "ubuntu2004"] -%}
```

Comment:
Should we only keep 1 ubuntu OS here?

Reply:
Absolutely yes! I left Ubuntu1804; let's talk if you think we should replace it with Ubuntu2004

Reply:
I think the name of the variable highlight we should have a single os per distro, I'd remove ubuntu1804

Reply:
I removed Ubuntu2004 to avoid disrupting the existing set; let's talk if we prefer Ubuntu20